.circleci/config.yml

```diff
@@ -171,3 +181,13 @@ workflows:
             branches:
               ignore:
                 - master
+      - push-lambda:
+          file_dir: commit/${CIRCLE_SHA1}/
```

Comment:
@stylianosrigas Do we have plan to clean up those zip files that will remain there?

Reply:
@angeloskyratzakos not at the moment but I will create a ticket to sort this out.